sodacitylabs/boring-framework
# chore: NounHelper for resource, model etc. name forms

---
name: Pull Request
about: Ask for code to be merged into master
---

**Describe your Code**
Refactors the internal workings for getting the various singular, plural forms for resources, models, etc.

**Expected behavior**
No changes

**Related Issues**
closes #7 

**How to test**
Steps to test and verify the code works as expected:
1. Run the test suite
2. Run through getting started
